core/modules/utils/utils.js

```diff
@@ -685,19 +685,9 @@ exports.escapeRegExp = function(s) {
     return s.replace(/[\-\/\\\^\$\*\+\?\.\(\)\|\[\]\{\}]/g, '\\$&');
 };
 
-/*
-Extended version of encodeURIComponent that encodes additional characters including
-those that are illegal within filepaths on various platforms including Windows
-*/
-exports.encodeURIComponentExtended = function(s) {
-	return encodeURIComponent(s).replace(/[!'()*]/g,function(c) {
-		return "%" + c.charCodeAt(0).toString(16).toUpperCase();
-	});
-};
-
 // Checks whether a link target is external, i.e. not a tiddler title
 exports.isLinkExternal = function(to) {
-	var externalRegExp = /^(?:file|http|https|mailto|ftp|irc|news|obsidian|data|skype):[^\s<>{}\[\]`|"\\^]+(?:\/|\b)/i;
+	var externalRegExp = /^(?:file|http|https|mailto|ftp|irc|news|data|skype):[^\s<>{}\[\]`|"\\^]+(?:\/|\b)/i;
 	return externalRegExp.test(to);
```

Comment:
is this an intentional change in this PR?

Reply:
No, something went definitely wrong during transfer on my end. Pushed a fix already.

Reply:
Yes, you may be right. Probably my GitHub Desktop was not yet fully synced.